src/csharp/Microsoft.Spark/Broadcast.cs

```diff
@@ -171,11 +172,26 @@ private JvmObjectReference CreateBroadcast(SparkContext sc, T value)
             // Spark versions.
             bool encryptionEnabled = bool.Parse(
                 sc.GetConf().Get("spark.io.encryption.enabled", "false"));
+            JvmObjectReference _pythonBroadcast;
+            bool isDatabricks =
+            !string.IsNullOrEmpty(GetEnvironmentVariable("DATABRICKS_RUNTIME_VERSION"));
```

Comment:
Why not use `SettingUtils.IsDatabricks` ?

Reply:
Because it belongs to `Microsoft.Spark.Worker.Utils`, which would have to be included in this project just for this util, wasn't sure if we wanted to do that?

Reply:
Then let's move it to the `Microsoft.Spark` project.

Reply:
Moved `IsDatabricks` to `ConfigurationService`.